latest/index.html

```diff
@@ -455,6 +455,17 @@ <h3>Cross-Origin Access</h3>
            enables older web-based clients to access the service from a different domain.
         </p>
       </section>
+      <section>
+        <h3>Language Selection</h3>
+        <p>
+           Following [[RFC9110]], services SHOULD use the <code>Accept-Language</code> header to let clients select the language in which user-facing text is returned.
+           They SHOULD use the <code>Content-Language</code> in their responses to expose the language each response is returned in.
+        </p>
+        <p>
+          Examples of user-facing text in service responses are: the name of the service and the name of property configuration fields in the manifest, the name and description of entities, types and properties,
+          the contents of the entity preview pages.
```

Comment:
This should probably also control what URL is returned for the `documentation` URL in the manifest (unless some other mechanism is proposed for multilingual documentation).

Reply:
How about the same mechanism? The documentation URL itself could already react to `Accept-Language`, no? Isn't that how most websites (should) work?

Reply:
That sounds fine.

Reply:
So should we add the documentation here, like this:

```suggestion
          the contents of the entity preview pages, and the documentation linked in the manifest.
```